lib/src/file_picker.dart

```diff
@@ -157,6 +157,10 @@ abstract class FilePicker extends PlatformInterface {
   /// For mobile platforms, this function will save file with [bytes] to return a path.
   /// Throws UnsupportedError on macOS if bytes are provided.
   ///
+  /// On the web, this function will start a download for the file with [bytes] and [fileName].
```

Comment:
The "throws an ArgumentError" section is listed twice?

```suggestion
  /// On the web, this function will start a download for the file with [bytes] and [fileName].
  /// If the [bytes] or [fileName] are omitted, this will throw an [ArgumentError].
  ///
```